platforms/chibios/drivers/audio_pwm_hardware.c

```diff
@@ -31,6 +32,20 @@
 extern bool    playing_note;
 extern bool    playing_melody;
 extern uint8_t note_timbre;
+static bool channel_1_stopped = true;
```

Comment:
After removing the (invalid) code to start the timer from `channel_1_set_frequency()` this variable becomes write-only and can be removed.
```suggestion
```